sklearn/multiclass.py

```diff
@@ -509,16 +525,19 @@ def partial_fit(self, X, y, classes=None):
         if _check_partial_fit_first_call(self, classes):
             self.estimators_ = [clone(self.estimator) for i in
                                 range(self.n_classes_ *
-                                (self.n_classes_-1) // 2)]
+                                      (self.n_classes_ - 1) // 2)]
 
         X, y = check_X_y(X, y, accept_sparse=['csr', 'csc'])
         check_classification_targets(y)
-        self.estimators_ = Parallel(n_jobs=self.n_jobs)(
-            delayed(_partial_fit_ovo_binary)(
-                estimator, X, y, self.classes_[i], self.classes_[j])
-            for estimator, (i, j) in izip(self.estimators_, ((i, j) for i
-                                in range(self.n_classes_) for j in range
-                                            (i + 1, self.n_classes_))))
+        self.estimators_ = Parallel(
+            n_jobs=self.n_jobs)(
+                delayed(_partial_fit_ovo_binary)(
+                    estimator, X, y, self.classes_[i], self.classes_[j])
+                for estimator, (i, j) in izip(
+                        self.estimators_, ((i, j)
```

Comment:
For cosmetic reasons, can you pull the latter generator expression out as a separate variable `combinations`?

In fact, it can be set as `combinations = itertools.combinations(range(self.n_classes), 2)`